news/2023/2023-10-22-halloween-descends.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+title: "Halloween 2023 Descends"
+date: 2023-10-22 14:00:00 +0000
+---
+
+By the ineffable will of the Dean "peppy" Herbert, we are having Halloween right this very moment — so come check out the winners from the Halloween 2023 Fanart Contest, soon to be featured in a game client near you!
+
+![](https://assets.ppy.sh/contests/186/banners/mocapixel.jpg)
+
+We've known this for a while, but we think you guys might have a bit of a thing for spooky month (and also possibly witches). Over **178** entries went up for a harrowing round of voting across almost a week, and the osu! community leapt so hard into the voting that we almost thought things were broken for a second or two.
+
+But with only **20** votes each to go around, the bulk of us truly felt the greatest fear Halloween truly has to offer: too many damn good choices, and not enough votes to spend for them all. It's like having an entire candy basket full of every particular treat you like — where do you even start?!
+
+Somehow, you managed, and we now have our latest Halloween 2023 seasonal backgrounds for all to behold. But enough waffling, it's time to check out the winners:
+
+## Finalists
+
+[![](https://assets.ppy.sh/contests/186/banners/mocapixel.jpg)](https://assets.ppy.sh/contests/186/winners/mocapixel.png)
+
+**2420** voters fell prey to the enchanting charms of contest veteran [**mocapixel**](https://osu.ppy.sh/users/11596939)'s witch pippi! It was clear from the opening hours of the contest that this piece was going to hold a lead — and *hold it did*, with hundreds of votes practically the entire time. A stunning combination of colour work and magical magnificence really captures that quintessential Halloween feeling, all while remaining easy to look at.
+
+Speaking of charms, the 4th time is it for mocapixel, who earns their first top-voted finalist place with flying colours! Quite literally, in this case, because you know, witches, brooms, so on... ahem. Congratulations!
```

Comment:
i dunno about this grammar

```suggestion
Speaking of charms, the 4th time it would become for mocapixel, earning their first top-voted finalist place with flying colours! Quite literally, in this case, because you know, witches, brooms, so on... ahem. Congratulations!
```

Reply:
would it be better to repeat something like

Speaking of charms, the 4th time really is the charm for mocapixel...

Reply:
oh upon another reread i finally figured out how this is supposed to be parsed

```suggestion
Speaking of charms, the 4th time turned out to be it for mocapixel, who earns their first top-voted finalist place with flying colours! Quite literally, in this case, because you know, witches, brooms, so on... ahem. Congratulations!
```